FilenameBear.py: Add ``spacecase`` convention

amrandahvur opened the pull request:
Add ``spacecase`` convention for Bear to recognize it

Closes https://github.com/coala/coala-bears/issues/556

Adrianzatreanu commented:
hey, could you please add the tests in the same commit? simply squash them while rebasing :D

amrandahvur commented on tests/general/FilenameBearTest.py:
```diff
@@ -45,6 +45,13 @@ def test_pascal_case(self):
+        self.check_validity(self.uut, [""], filename="/Home/xyz/x_y.py",
+                            valid=False)
+        self.check_validity(self.uut, [""], filename="XYZ/__Init__.py")
+        self.check_validity(self.uut, [""], filename="/a/camCase")
```
Sure, Working on it :D

amrandahvur commented:
@Adrianzatreanu I have squashed and rebased them, Is the new commit message ok ? Considering the shortlog contains the name of only "FilenameBear.py" and not "FilenameBearTest.py" ??
@Makman2 I have written them, Do I need to add more test cases ??

amrandahvur commented:
@Nosferatul Done! Please Check ?

Nosferatul commented:
It's ok, just wait for @Makman2 to see if more tests are required

sils commented:
ack https://github.com/coala/coala-bears/pull/916/commits/dd82875732853bd2ce4c9c364eb81abddddcc0aa tests are sufficient - the spacecase function tests are more elaborate and shouldn't be duplicated here so it's fine, @Nosferatul why did you mark it WIP? WIP means the author has to change something

sils commented:
@rultor merge

rultor commented:
> @rultor merge

@sils OK, I'll try to merge now. You can check the progress of the merge [here](http://www.rultor.com/t/9812-255213276)

rultor commented:
> @rultor merge

@sils Done! FYI, the full log is [here](http://www.rultor.com/t/9812-255213276) (took me 1min)

amrandahvur commented:
@sils This was my first "difficulty/low" issue. What else do I need to do to become a member ??

Nosferatul commented:
@sils I didn't know if the tests are sufficient. My bad 😟
